ext/lb/lb.go

```diff
@@ -298,12 +299,27 @@ func (l *LoadBalancer) Name() string {
 	return pluginName
 }
 
-func (l *LoadBalancer) ProxyContainers(name string) ([]dockerclient.Container, error) {
-	containers, err := l.client.ListContainers(false, false, "")
+
+func (l *LoadBalancer) ListContainers() ([]dockerclient.Container, error) {
+	var containers []dockerclient.Container
+	var err error
+
+	allContainers, err := l.client.ListContainers(false, false, "")
```

Comment:
This was recently updated to list all containers to make sure to grab proxy containers that were stopped previously (i.e. in the event of an error in the config or something).